doc/specs/lichess-api.yaml

```diff
@@ -4,7 +4,6 @@ info:
   title: Lichess.org API reference
   x-logo:
     url: https://lichess1.org/assets/logo/lichess-pad12.svg
-    backgroundColor: #ffffff
```

Comment:
This doesn't actually work. `#ffffff` is a comment. It could be put in quotes but I don't think we actually want a white background for the logo. Keeping the gray background from the sidebar as it is currently seems better.